test/tla/Paxos.tla

```diff
@@ -124,8 +129,13 @@ TypeOK == /\ maxBal \in [Acceptor -> Ballot \cup {-1}]
 (* the array `votes' describing the votes cast by the acceptors is defined *)
 (* as follows.                                                             *)
 (***************************************************************************)
+
+\* the original specification does not have this definition, we need it for types
```

Comment:
Why do we need this? Is this kind of thing a short term work around, or a permanent requirement?

Reply:
I think it is a permanent construct. It is often too cumbersome to distinguish between tuples and sequences. I was even thinking of introducing operators like in Scala: `tuple0()`, `tuple1(x)`, `tuple2(x, y)`, ...